tree/20_cfe_basics/cfengine/paths.cf

```diff
@@ -215,6 +215,8 @@ bundle common paths
       "path[sort]"     string => "/usr/bin/sort";
       "path[test]"     string => "/usr/bin/test";
       "path[wget]"     string => "/usr/bin/wget";
+      "path[augtool]"  string => "/usr/bin/augtool";
```

Comment:
should be in ` tree/20_cfe_basics/paths.cf`. The `cfengine` folder is what comes from the upstream cfengine project.